tests/math/unary/test_nondifferentiable.py

```diff
@@ -51,3 +51,23 @@ def test_argmax(a, data):
 
     # array input
     assert_allclose(mg.argmax(a, axis=axis), np.argmax(a, axis=axis))
+
+
+@given(
+    a=hnp.arrays(
+        shape=hnp.array_shapes(max_side=4, max_dims=5), dtype=dtype_strat_numpy
+    ),
+    data=st.data(),
+)
+def test_any(a, data):
+    axis = data.draw(valid_axes(ndim=a.ndim), label="axis")
+    tensor = Tensor(a)
+
+    # tensor input
+    assert_allclose(mg.any(tensor, axis=axis), np.any(a, axis=axis))
+
+    # tensor method
+    assert_allclose(tensor.any(axis=axis), a.any(axis=axis))
+
+    # array input
+    assert_allclose(mg.any(a, axis=axis), np.any(a, axis=axis))
```

Comment:
`keepdims` also needs to be exercised in these tests - this parameter should also be drawn by Hypothesis.

Reply:
This should be resolved - thanks!